docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '2'
```

Comment:
What's the significance of this version? Can I just set it to the package version?